infra/test/integration/simple_example/simple_example_test.go

```diff
@@ -63,7 +63,7 @@ func TestSimpleExample(t *testing.T) {
 			}
 			return false, nil
 		}
-		utils.Poll(t, isBlueMigServing, 20, time.Second*20)
+		utils.Poll(t, isBlueMigServing, 10, time.Minute*2)
```

Comment:
These changes don't seem related to the architecture diagram / README :)

What does this diff do?

Reply:
I was troubleshooting the failing CFT tests thinking it was related to timeouts, but it appears not to be.  I'll roll back.